core/src/datayoga_core/job.py

```diff
@@ -127,6 +127,8 @@ def transform(self, data: List[Dict[str, Any]], deepcopy: boolean = True) -> Job
 
     async def run(self):
         async for records in self.producer.produce():
+            prom.incoming_records.inc(len(records))
```

Comment:
Does it make sense to do it always even if Prometheus is not started?

Reply:
It just increment a value... I don't think we're buying time here if we add condition.